.gitpod/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM gitpod/workspace-java-17
+
+ENV PATH="/home/gitpod/bin:/workspace/clojure-polylith-realworld-example-app/.gitpod:${PATH}"
+ENV JAVA_HOME="/home/gitpod/.sdkman/candidates/java/current"
+ENV JAVA_TOOL_OPTIONS="-Xmx3489m"
+
+RUN mkdir /home/gitpod/bin
+RUN wget -P /home/gitpod https://github.com/polyfy/polylith/releases/download/v0.2.17-alpha/poly-0.2.17-alpha.jar
+RUN bash -c cat <<'EOF' > /home/gitpod/bin/poly && chmod +x /home/gitpod/bin/poly
+#!/bin/bash
+
+java $JVM_OPTS "-jar" "/home/gitpod/poly-0.2.17-alpha.jar" $@
```

```suggestion
RUN wget -P /home/gitpod https://github.com/polyfy/polylith/releases/download/stable-master/poly-0.2.18-SNAPSHOT.jar
RUN bash -c cat <<'EOF' > /home/gitpod/bin/poly && chmod +x /home/gitpod/bin/poly
#!/bin/bash

java $JVM_OPTS "-jar" "/home/gitpod/poly-0.2.18-SNAPSHOT.jar" $@
```

Should we use the latest version from the master branch of the Polylith repository instead? What do you think @tengstrand?

---

I tried this on the master branch of my fork now. It works great. (As if you doubted, but anyway, an info point for your decision, @tengstrand ).

---

Yes, let's use `0.2.18-SNAPSHOT`!